subworkflows/local/bgc.nf

```diff
@@ -0,0 +1,68 @@
+/*
+    Run AMP screening tools
+*/
+include { UNTAR as UNTAR_CSS                       } from '../../modules/nf-core/modules/untar/main'
+include { UNTAR as UNTAR_DETECTION                 } from '../../modules/nf-core/modules/untar/main'
+include { UNTAR as UNTAR_MODULES                   } from '../../modules/nf-core/modules/untar/main'
+include { ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES } from '../../modules/nf-core/modules/antismash/antismashlitedownloaddatabases/main'
+include { ANTISMASH_ANTISMASHLITE                  } from '../../modules/nf-core/modules/antismash/antismashlite/main'
+
+workflow BGC {
+
+    take:
+    fna     // tuple val(meta), path(PROKKA.out.fna)
+    gff     // tuple val(meta), path(PROKKA.out.gff)
+
+    main:
+    ch_versions = Channel.empty()
+
+    // Check whether user supplies database and/or antismash directory. If not, obtain them via nf-core/modules/antismashlite/antismashlitedownloaddatabases.
+    // Important for future maintenance: For CI tests, only the "else" option below is used. Both options should be tested locally whenever the antiSMASH module gets updated.
+    if ( params.bgc_antismash_databases && params.bgc_antismash_directory ) {
+
+        // Supply databases
+        ch_antismash_databases = Channel
+            .fromPath( params.bgc_antismash_databases )
+            .first()
+
+        // Supply antiSMASH directory
+        ch_antismash_directory = Channel
+            .fromPath( params.bgc_antismash_directory )
+            .first()
+
+    } else {
+
+        print "Warning: No antiSMASH database and/or directory supplied – they will be downloaded by the pipeline."
```

Comment:
```suggestion
        print "Warning: No antiSMASH database and/or directory supplied – they will be downloaded by the pipeline."
```

Try using `log.warn` instead, this should pretty print

Also when printing in groovy, I recommend `println` instead, to make sure there is a new line at the end of the string